bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/AbstractHomeConnectThingHandler.java

```diff
@@ -1172,6 +1172,14 @@ protected void handleLightCommands(final ChannelUID channelUID, final Command co
         }
     }
 
+    protected void handlePowerCommand(final ChannelUID channelUID, final Command command,
+            final HomeConnectApiClient apiClient, String stateNotOn)
```

Comment:
Good idea. Should work for all appliances.
![Bildschirmfoto-20210527175827-1142x240](https://user-images.githubusercontent.com/2088078/119861230-71d6fd00-bf17-11eb-94e3-cd44fc26dce4.png)